src/app/program/program.component.ts

```diff
@@ -26,7 +26,7 @@ export class ProgramComponent implements OnInit, AfterViewInit {
     subject!: Observable<SidebarElement[]>;
     nameFormControl: FormControl = new FormControl("");
     program: Program | undefined;
-
+    isButtonActive: boolean = false;
```

Comment:
what is the purpose of all of these 'isButtonActive' variables? Their values never get reassigned after initialization, so I think they can be removed. This pertains to program-component, voice-assistant-component and voice-assistant-chat-component